challenge.rb

```diff
@@ -1,26 +1,25 @@
-
 def capitalize_each_string(input)
-  #implement your solution here
+  return input.map{ |animal| animal.capitalize}
```

Comment:
explicit returns are not necessary in ruby. Look up implicit returns if you want to know more.